src/Components/Question/Question.js

```diff
@@ -2,9 +2,9 @@ import React, { useState, useEffect } from "react";
 import "./Question.css"
 import Audio from '../Audio/Audio'
 import Choices from "../Choices/Choices";
-import EndGame from "../EndGame/EndGame";
-import Loading from "../Loading/Loading";
-
+import Endgame from "../Endgame/Endgame";
+import Loading from "../Loading/Loading"
```

Comment:
Can we add a semi-colon to the end of line 6 for consistency?  Also as a general note,  we should go through and just make sure our formatting is the same after code freeze.